TurkishFilter/sections/css_extended.txt

```diff
@@ -263,6 +263,7 @@ forum.miuiturkiye.net#?#.mainContent > div[class="sectionMain funbox"]:has(>.fun
 ozengen.com#?#.code-block:has(>.adsbygoogle)
 ozengen.com#?#.sidebar-narrow > .widget > .widget-top:has(>h4:contains(Sponsorlu Bağlantılar))
 donanimhaber.com#?#.eng_m_asw > .eng_m_widget_sw:has(>.eng_ads)
+donanimhaber.com##.arkaplan-giydirme
```

Comment:
Please move this code into `TurkishFilter/sections/specific.txt`.

Reply:
Oh, this line already exists there, sorry.
I am closing the PR.